pages/signup.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FueledUp | Home</title>
+
+    
+    <!-- PowertrainUI library -->
+    <link
+    rel="stylesheet"
+    href="https://powertrain-ui.netlify.app/styles.css"
+    />
+    
+    <!-- Page CSS -->
+    <link rel="stylesheet" href="../css/signup.css" />
+    
+  </head>
+  <body>
+    <div class="wrapper">
+      <div class="header pos-st top-left-pos py-md">
+        <header class="max-width-1200 nav-main flex-row px-md mx-auto">
+          <div class="nav-logo">
+            <a href="../../index.html">FueledUp</a>
+          </div>
+          <div class="search-input">
+            <div class="fas fa-search search-icon"></div>
+            <label for="searchbar"></label>
+            <input
+              class="input-round input-md border-2"
+              type="text"
```

Comment:
Suggestion: input of **type = "search"** can be used instead of text for search bars as a best practice.

Reply:
Thank you for pointing this out! Learnt something new today. Have made the suggested changes!